Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -140,6 +136,17 @@ open class MediaAttachment: NSTextAttachment {
     }
 
 
+    // MARK: - Open Helpers
```

Comment:
This category name is very generic and of no use.  Why not switch it to something more specific like "Overlay support"?